src/NosCore.WorldServer/Controllers/RelationsController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using NosCore.Core;
+using NosCore.Core.Serializing;
+using NosCore.Data.WebApi;
+using NosCore.GameObject.Networking;
+using NosCore.Shared.Enumerations.Account;
+using NosCore.Shared.Enumerations.Interaction;
+
+namespace NosCore.WorldServer.Controllers
+{
+    [Route("api/[controller]")]
+    [AuthorizeRole(AuthorityType.GameMaster)]
+    public class RelationsController : Controller
+    {
+        // POST api/relations
+        [HttpPost]
+        public IActionResult PostUpdateFriendList([FromBody] PostedPacket postedPacket)
```

Comment:
this is not a post you are trying to post a packet it has nothing to do here use packet api